apps/src/code-studio/pd/workshop_dashboard/local_summer_workshop_survey_results.jsx

```diff
@@ -102,6 +116,43 @@ const LocalSummerWorkshopSurveyResults = React.createClass({
     );
   },
 
+  renderRow(row, i) {
+    let scoreCells;
+    let thisWorkshopData = this.state.thisWorkshop[row['key']];
+
+    if (this.state.facilitatorBreakdown && typeof thisWorkshopData === 'object') {
+      scoreCells = this.state.facilitatorNames.map((facilitator, i) => {
+        return (
+          <td key={i}>
+            {this.renderScore(row, thisWorkshopData[facilitator])}
+          </td>
+        );
+      });
+
+      scoreCells.unshift((<td key={this.state.facilitatorNames.length}/>));
+    } else {
+      scoreCells = [(
+        <td key={0}>
+          {this.renderScore(row, thisWorkshopData)}
+        </td>
+      )];
+
+      if (this.state.facilitatorBreakdown) {
+        _.times(this.state.facilitatorNames.length, (i) => {
+          scoreCells.push((<td key={i + 1}/>));
+        });
```

Comment:
If thisWorkshopData is an object, that means that the question is facilitator specific, and is a hash of facilitator names to average scores, and the scores need to be displayed in the facilitator cells.

Line 132 adds the cell that goes under the "This workshop" column for facilitator specific questions. The key is arbitrary, but I can make it something better. However, keys are mandatory.